app/assets/stylesheets/application.css

```diff
@@ -21,100 +10,35 @@ h3 {
   padding-top: 10px;
   padding-left: 5px;
   padding-bottom: 0;
-  font-family: 'Courier New', Courier, monospace;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
   text-decoration: none;
 }
 
-.inputmail {
-  background-color: #f6f6f6;
-  border: none;
-  color: #0d0d0d;
-  padding: 10px 25px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  margin: 5px;
-  width: 50%;
-  border-radius: 5px 5px 5px 5px;
-  box-shadow: 2px 2px 3px #000000;
-}
-
-.inputmail:hover {
-  border-bottom: 3px solid black;
-  
-}
-
-.box {
-  background-color: #ece4e4;
-  border: none;
-  color: #0d0d0d;
-  padding: 10px 50px;
-  text-align: left;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 15px;
-  margin: 5px;
-  width: 50%;
-  height: 150px;
-  border-radius: 20px 20px 20px 20px;
-}
-
-.box:hover {
-  box-shadow:0px 0px 0px -1px rgba(0, 0, 0, 0.2), 0px 0px 10px 0px rgba(0, 0, 0, 0.19);
-  z-index:1;
-}
-
-button, .btn {
-  border-radius: 80px;
-  margin: 20px;
-  background-color: black;
-  border: none;
-  text-align: center;
-  text-decoration: none;
-  padding: 5px 30px;
-  color: white;
-  font-size: 18px;
-  box-shadow: 2px 2px 4px #000000;
-  
-}
-.btn:hover {
-  color: white;
-}
-
 .msg {
```

Comment:
remove this class altogether and usage also.

Reply:
reduced max. custom css.